lib/rest_6.js

```diff
@@ -798,21 +798,21 @@ function* getChainInfos(chainIds, node) {
  * @param{Array} member initial balance
  * @param{String} contract string
  * @param{Object} contract instance variables
- * @param{String} governance contract name
- * @param{Number} node target node
+ * @param{Object} options flags for history and indexing
  * @returns{()} ChainId
  */
-function* createChain(label, members, balances, src, args, contract, node) {
-  verbose('createChain', {label, members, balances, src, args, contract, node});
+function* createChain(label, members, balances, src, args, options={}) {
```

Comment:
This is a breaking change, can you confirm that this function isn't being used by any client applications?